VAMobile/src/screens/HomeScreen/ProfileScreen/ContactInformationScreen/ContactInformationScreen.tsx

```diff
@@ -89,7 +89,7 @@ const ContactInformationScreen: FC<ContactInformationScreenProps> = ({ navigatio
   useFocusEffect(
     React.useCallback(() => {
       if (needsDataLoad && profileNotInDowntime) {
-        dispatch(getProfileInfo(ScreenIDTypesConstants.PERSONAL_INFORMATION_SCREEN_ID))
+        dispatch(getProfileInfo(ScreenIDTypesConstants.CONTACT_INFORMATION_SCREEN_ID))
```

Comment:
Changing this because I'm making it more screen id driven and this felt like it was given the wrong screen id in order to use another screen's values.